### PR TITLE
[WIP]buy_index-edit1

### PR DESCRIPTION
**Pull request by melaleyraud6**
# What  
 商品購入ページのビュー実装をしました

# Why
　商品購入に必要だからです

### CODE REVIEW

File: app/helpers/buys_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module BuysHelper
```
Comment (melaleyraud6) [inline app/helpers/buys_helper.rb]: > __[Rails Best Practices]__ remove empty helpers ([see on Sider](https://sider.review/gh/repos/225556309/pulls/17#issue-4919067))
>
> <details>
>
> | Rule |
> | - |
> | RailsBestPractices::Reviews::RemoveEmptyHelpersReview |
>
> References:
> - <https://rails-bestpractices.com/posts/2011/04/09/remove-empty-helpers/>
>
> </details>

ファイルを削除しました

File: app/assets/stylesheets/buys.scss [hunk 2]
```diff
@@ -0,0 +1,180 @@
+
+.buy-index__single-wrapper {
+  background-color: #f5f5f5;
+  width: 100vw;
+  font-family: 'Source Sans Pro', Helvetica , Arial, 'æ¸¸ã‚´ã‚·ãƒƒã‚¯ä½“', 'YuGothic', 'ãƒ¡ã‚¤ãƒªã‚ª', 'Meiryo', sans-serif;
+  header__single-wrapper {
```
Comment (melaleyraud6) [inline app/assets/stylesheets/buys.scss]: > __[stylelint]__ Unexpected unknown type selector "header__single-wrapper" ([see on Sider](https://sider.review/gh/repos/225556309/pulls/17#issue-4919061))
>
> <details>
>
> | Rule | Severity |
> | - | - |
> | selector-type-no-unknown | error |
>
> References:
> - <https://github.com/stylelint/stylelint/tree/12.0.0/lib/rules/selector-type-no-unknown>
>
> </details>

クラス名を書き換えました

File: config/routes.rb [hunk 3]
```diff
@@ -1,3 +1,9 @@
 Rails.application.routes.draw do
  root  'top#index'
+  resources :buys do
```
Comment (melaleyraud6) [inline config/routes.rb]: > __[Rails Best Practices]__ restrict auto-generated routes buys (only: [:show, :index]) ([see on Sider](https://sider.review/gh/repos/225556309/pulls/17#issue-4919068))
>
> <details>
>
> | Rule |
> | - |
> | RailsBestPractices::Reviews::RestrictAutoGeneratedRoutesReview |
>
> References:
> - <https://rails-bestpractices.com/posts/2011/08/19/restrict-auto-generated-routes/>
>
> </details>

ルーティングを書き直しました

File: app/assets/stylesheets/buys.scss [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+              }
+              h3 {
+                font-size: 14px;
+                padding:0;
+              }
+              .user-right {
+                color: #0099e8;
+              }
+              .item-info {
+                font-family: 'icon-font';
```
Comment (melaleyraud6) [inline app/assets/stylesheets/buys.scss]: > __[stylelint]__ Unexpected missing generic font family ([see on Sider](https://sider.review/gh/repos/225556309/pulls/17#issue-4919064))
>
> <details>
>
> | Rule | Severity |
> | - | - |
> | font-family-no-missing-generic-family-keyword | error |
>
> References:
> - <https://github.com/stylelint/stylelint/tree/12.0.0/lib/rules/font-family-no-missing-generic-family-keyword>
>
> </details>

削除しました

File: app/assets/stylesheets/buys.scss [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+      h2 {
+        font-size: 22px;
+        padding:32px;
+        text-align: center;
+        margin: 0;
+      }
+      .item-content:first-of-type {
+        margin: 0;
+      }
+      .item-content {
```
Comment (melaleyraud6) [inline app/assets/stylesheets/buys.scss]: > __[stylelint]__ Expected selector ".buy-index__single-wrapper .main-contents section .item-content" to come before selector ".buy-index__single-wrapper .main-contents section .item-content:first-of-type" ([see on Sider](https://sider.review/gh/repos/225556309/pulls/17#issue-4919062))
>
> <details>
>
> | Rule | Severity |
> | - | - |
> | no-descending-specificity | error |
>
> References:
> - <https://github.com/stylelint/stylelint/tree/12.0.0/lib/rules/no-descending-specificity>
>
> </details>

クラス名を変更しました

File: app/assets/stylesheets/buys.scss [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+               padding: 0;
+             }
+           }
+           .icon-text {
+            font-style: normal;
+           }
+          }
+        }
+      }
+      .item-content:last-of-type {
```
Comment (melaleyraud6) [inline app/assets/stylesheets/buys.scss]: > __[stylelint]__ Expected selector ".buy-index__single-wrapper .main-contents section .item-content:last-of-type" to come before selector ".buy-index__single-wrapper .main-contents section .item-content .item-content-inner form .item-content" ([see on Sider](https://sider.review/gh/repos/225556309/pulls/17#issue-4919066))
>
> <details>
>
> | Rule | Severity |
> | - | - |
> | no-descending-specificity | error |
>
> References:
> - <https://github.com/stylelint/stylelint/tree/12.0.0/lib/rules/no-descending-specificity>
>
> </details>

位置を書き換えました

File: app/assets/stylesheets/buys.scss [hunk 7]
```diff
@@ -0,0 +1,180 @@
...
+              width: 100%;
+              .buy-you-pay {
+                font-size: 30px;
+                display: table-row;
+                .item-price-cell:first-child {
+                  text-align: left;
+                  font-size:18px;
+                  font-weight: bold;
+                }
+                .item-price-cell {
```
Comment (melaleyraud6) [inline app/assets/stylesheets/buys.scss]: > __[stylelint]__ Expected selector ".buy-index__single-wrapper .main-contents section .item-content .item-content-inner form .item-price-table .buy-you-pay .item-price-cell" to come before selector ".buy-index__single-wrapper .main-contents section .item-content .item-content-inner form .item-price-table .buy-you-pay .item-price-cell:first-child" ([see on Sider](https://sider.review/gh/repos/225556309/pulls/17#issue-4919063))
>
> <details>
>
> | Rule | Severity |
> | - | - |
> | no-descending-specificity | error |
>
> References:
> - <https://github.com/stylelint/stylelint/tree/12.0.0/lib/rules/no-descending-specificity>
>
> </details>

書き直しました

File: app/assets/stylesheets/buys.scss [hunk 8]
```diff
@@ -0,0 +1,180 @@
...
+              }
+            }
+           .address-content {
+             display: flex;
+             justify-content: space-between;
+             h3 {
+               padding: 0;
+             }
+           }
+           .icon-text {
```
Comment (melaleyraud6) [inline app/assets/stylesheets/buys.scss]: > __[stylelint]__ Expected selector ".buy-index__single-wrapper .main-contents section .item-content .item-content-inner form .icon-text" to come before selector ".buy-index__single-wrapper .main-contents section .item-content .item-content-inner form .item-content .item-info .icon-text" ([see on Sider](https://sider.review/gh/repos/225556309/pulls/17#issue-4919065))
>
> <details>
>
> | Rule | Severity |
> | - | - |
> | no-descending-specificity | error |
>
> References:
> - <https://github.com/stylelint/stylelint/tree/12.0.0/lib/rules/no-descending-specificity>
>
> </details>

書き直しました

File: app/assets/stylesheets/buys.scss [hunk 9]
```diff
@@ -0,0 +1,180 @@
+
+.buy-index__single-wrapper {
+  background-color: #f5f5f5;
+  width: 100vw;
+  font-family: 'Source Sans Pro', Helvetica , Arial, 'æ¸¸ã‚´ã‚·ãƒƒã‚¯ä½“', 'YuGothic', 'ãƒ¡ã‚¤ãƒªã‚ª', 'Meiryo', sans-serif;
+  header-single-wrapper {
```
Comment (melaleyraud6) [inline app/assets/stylesheets/buys.scss]: > __[stylelint]__ Unexpected unknown type selector "header-single-wrapper" ([see on Sider](https://sider.review/gh/repos/225556309/pulls/17#issue-4921368))
>
> <details>
>
> | Rule | Severity |
> | - | - |
> | selector-type-no-unknown | error |
>
> References:
> - <https://github.com/stylelint/stylelint/tree/12.0.0/lib/rules/selector-type-no-unknown>
>
> </details>

書き直しました

File: app/assets/stylesheets/buys.scss [hunk 10]
```diff
@@ -0,0 +1,180 @@
...
+              width: 100%;
+              .buy__you-pay {
+                font-size: 30px;
+                display: table-row;
+                .item__price-cell:first-child {
+                  text-align: left;
+                  font-size:18px;
+                  font-weight: bold;
+                }
+                .item__price-cell {
```
Comment (melaleyraud6) [inline app/assets/stylesheets/buys.scss]: > __[stylelint]__ Expected selector ".buy-index__single-wrapper .main__contents section .item__content .item__content--inner form .item__price-table .buy__you-pay .item__price-cell" to come before selector ".buy-index__single-wrapper .main__contents section .item__content .item__content--inner form .item__price-table .buy__you-pay .item__price-cell:first-child" ([see on Sider](https://sider.review/gh/repos/225556309/pulls/17#issue-4921370))
>
> <details>
>
> | Rule | Severity |
> | - | - |
> | no-descending-specificity | error |
>
> References:
> - <https://github.com/stylelint/stylelint/tree/12.0.0/lib/rules/no-descending-specificity>
>
> </details>

書き直しました

Comment (shogo-naga8) [reply app/assets/stylesheets/buys.scss]: LGTM

File: config/routes.rb [hunk 11]
```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
   root  'items#index'
+  get  "/buys/index", to: "buys#index"
```
Comment (ken-nag) [inline config/routes.rb]: reosurceで書きましょう

File: config/initializers/carrierwave.rb [hunk 12]
```diff
@@ -2,16 +2,16 @@
...
+#     region: 'ap-northeast-1'
+#   }
 
-  config.fog_directory  = 'freemarketsample64-b'
-  config.asset_host = 'https://s3-ap-northeast-1.amazonaws.com/freemarketsample64-b'
-end
\ No newline at end of file
+#   config.fog_directory  = 'freemarketsample64-b'
+#   config.asset_host = 'https://s3-ap-northeast-1.amazonaws.com/freemarketsample64-b'
+# end
```
Comment (ken-nag) [inline config/initializers/carrierwave.rb]: 不要なら消しましょう

File: app/assets/stylesheets/buys.scss [hunk 13]
```diff
@@ -0,0 +1,176 @@
...
+            }
+            .item__point {
+              color:#888;
+              margin: 24px 0 32px;
+              padding-left: 0;
+              line-height: 27px;
+              .item__use-point {
+                text-align: left;
+                #checkbox-id {
+                  // display:none;
```
Comment (ken-nag) [inline app/assets/stylesheets/buys.scss]: 不要なコメントアウトは残さないようにしましょう

File: config/routes.rb (hunk 11 above)
Comment (melaleyraud6) [reply config/routes.rb]: ビューの画面を確認するためのルーティングです。

File: config/initializers/carrierwave.rb (hunk 12 above)
Comment (melaleyraud6) [reply config/initializers/carrierwave.rb]: コントローラー作成の時にエラーが起こったため、コメントアウトして作ったまま、そのままにしていました。
今はコメントアウトを取り消しています

File: app/assets/stylesheets/buys.scss (hunk 13 above)
Comment (melaleyraud6) [reply app/assets/stylesheets/buys.scss]: 不要なコメントアウトを削除しました

**Comment by i-date** [comment]
最新の修正分がまだgithubに反映されていないようです…。
githubの処理が遅いだけかもしれないですが、一度push忘れなどをご確認いただけないでしょうか？

**Comment by ken-nag** [comment]
```最新の修正分がまだgithubに反映されていないようです…。
githubの処理が遅いだけかもしれないですが、一度push忘れなどをご確認いただけないでしょうか？```
このコメントの後にcommitなどのアクションがなさそうです、もう一度pushするなど確認お願いします

**Comment by shingoteshima** [comment]
LGTMです！

**Pull request merged**